### PR TITLE
Update git / workflow / branches

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Change Link Name

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/git/workflow/branches/viewing-your-tracking-branches.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/git/workflow/branches/viewing-your-tracking-branches.md?ref=a7750d5b8546ffdd14ebca1610a811e19e4adaed)

File: git/workflow/branches/checkout-the-previous-branch.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ aspects:
 
 links:
 
-  - '[github.com](https://github.com/jbranchaud/til/blob/master/git/checkout-previous-branch.md){website}'
+  - '[Checkout Previous Branch Github.com](https://github.com/jbranchaud/til/blob/master/git/checkout-previous-branch.md){website}'
```
Comment (kapnobatai137) [inline git/workflow/branches/checkout-the-previous-branch.md]:
```suggestion
  - '[Checkout Previous Branch Github](https://github.com/jbranchaud/til/blob/master/git/checkout-previous-branch.md){website}'
```

File: git/workflow/branches/move-the-last-commit-to-a-new-branch.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ aspects:
 
 links:
 
-  - '[git-scm.com](https://git-scm.com/docs/git-reset){website}'
+  - '[Git Reset Official Documentation](https://git-scm.com/docs/git-reset){website}'
```
Comment (kapnobatai137) [inline git/workflow/branches/move-the-last-commit-to-a-new-branch.md]:
```suggestion
  - '[Git Reset](https://git-scm.com/docs/git-reset){documentation}'
```

File: git/workflow/branches/viewing-your-tracking-branches.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ aspects:
 
 links:
 
-  - '[git-scm.com](http://git-scm.com/book/en/v2/Git-Branching-Remote-Branches){website}'
+  - '[Tracking Branches Official Documentation](http://git-scm.com/book/en/v2/Git-Branching-Remote-Branches){website}'
```
Comment (kapnobatai137) [inline git/workflow/branches/viewing-your-tracking-branches.md]:
```suggestion
  - '[Tracking Branches](http://git-scm.com/book/en/v2/Git-Branching-Remote-Branches){documentation}'
```

**Pull request merged**